### DISCUSSION TITLE
希望提供移动端组件的演示和文档

### DISCUSSION THREAD

**Discussion by kagol** [opentiny/tiny-vue#114] [User voice]
用户声音来源：[https://www.cnblogs.com/kagol/p/17294802.html#5165538](https://www.cnblogs.com/kagol/p/17294802.html#5165538)

2023-04-07 09:06 [郑延杰](https://www.cnblogs.com/ZhyjEye/)
大佬，官网没有看到怎么跨端的演示，即使在移动端打开，也是PC的样式，是我打开的方式不对么

**Comment by qq1037305420** [comment]
+1 playground 有传入 mode 但是也不管用

**Comment by kagol** [comment]
移动端官网计划年后部署，移动端组件可以使用，使用方式有两种：
1. 全局使用方式
2. 单组件使用方式

## 全局使用方式

可以通过设置全局变量的方式使用 mobile 组件。

如果你是 Vue3 项目：
```
import { getCurrentInstance } from 'vue'

const app = getCurrentInstance()

app.config.globalProperties.tiny_mode = 'mobile'
```

如果你是 Vue2 项目：
```
Vue.prototype.tiny_mode = 'mobile'
```

## 单组件使用方式

```
<tiny-button tiny_mode='mobile'>移动端按钮</tiny-button>
```
